datasets/scottish-lidar.yaml

```diff
@@ -0,0 +1,43 @@
+Name: LiDAR (Light Detection and Ranging) for Scotland
+Description: This dataset is Lidar data that has been collected by the Scottish public sector and made available under the Open Government Licence. The data are DTM, DSM and LAZ LIDAR data types. The dataset contains multiple subsets of data which were each commissioned and flown in response to different organisational requirements. The details of each can be found at https://remotesensingdata.gov.scot/data#/list
+Documentation: https://remotesensingdata.gov.scot/data#/list
+Contact: https://remotesensingdata.gov.scot/feedback or email Scottish Government on gi-sat@gov.scot
+ManagedBy: Joint Nature Conservation Committee (JNCC)
+UpdateFrequency: New datasets have historically been added every 2-3 years but there is no guarantee that this will continue.
+Tags:
+  - lidar
+  - cities
+  - coastal
+  - environmental
+  - urban
+  - elevation
+License: "All data is made available under the [Open Government Licence v3](http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/) unless otherwise stated. Currently the exception is only the phase 2 - LAZ files which are under [non-commercial government license](https://www.nationalarchives.gov.uk/doc/non-commercial-government-licence/version/2/)."
+Resources:
+  - Description: LiDAR data (DSM, DTM and Laz)
+    ARN: arn:aws:s3:::srsp-open-data
+    Region: eu-west-2
+    Type: S3 Bucket
+DataAtWork:
+  Tutorials:
+    - Title: LiDAR Tutorial using QGIS
+      URL: https://datasciencescotland.github.io/LiDAR-Tutorial-QGIS/
+      AuthorName: Michal Michalski
+    - Title: LiDAR Tutorial using R
+      URL: https://datasciencescotland.github.io/LiDAR-Tutorial-R/
+      AuthorName: Michal Michalski
+  Tools & Applications:
+    - Title: Scottish Remote Sensing Portal
+      URL: https://remotesensingdata.gov.scot/

```

Comment:
@foxfoxfox7 can you add AuthorName/URL and then resubmit your PR (verify if correct also). It is required for dataset onboarding.
```
AuthorName: Scottish Government and Joint Nature Conservation Committee (JNCC)
AuthorURL: https://remotesensingdata.gov.scot/about
```